### ISSUE TITLE
Points requested in calculate_wake are not used

### ISSUE THREAD

**Issue by ewquon** [NREL/floris#62]
**Bug description**
Calling `fi.calculate_wake(points=my_points)` returns `fi.floris.farm.flow_field.*` that are not at the requested coordinates--the number of flowfield points differs from `my_points`. This is unexpected behavior. 

**Floris Version**
commit 2833cc7bc952aa7c5d66963a749ad71900d345ab

**System Information (please complete the following information):** 
Mac High Sierra

**Additional context**
<Add any other context about the problem here.>

**Comment by ewquon** [comment]
Example:
```python
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

import floris.tools as wfct
from floris.utilities import Vec3 # to specify grid resolution

#===============================
# sampling parameters
D = 126.0
ds = 0.1 * D
x1 = np.arange(-3*D,800+10*D,ds)
y1 = np.arange(-3*D,630+3*D,ds)
z1 = np.arange(0,3*D,ds)
#===============================

fi = wfct.floris_interface.FlorisInterface("example_input.json")

x,y,z = np.meshgrid(x1,y1,z1,indexing='ij')  # expected output

# THIS WORKS:
print('reinit with res and bounds')
fi.floris.farm.flow_field.reinitialize_flow_field(
    with_resolution=Vec3(len(x1),len(y1),len(z1)), # why Vec3?
    bounds_to_set=[x1[0],x1[-1],y1[0],y1[-1],z1[0],z1[-1]]
)
fi.calculate_wake()
assert np.allclose(fi.floris.farm.flow_field.x, x)
assert np.allclose(fi.floris.farm.flow_field.y, y)
assert np.allclose(fi.floris.farm.flow_field.z, z)

# THIS DOES NOT WORK:
print('calc with pts')
points = np.stack((x.ravel(),y.ravel(),z.ravel()), axis=0)
fi.calculate_wake(points=points)
assert np.allclose(fi.floris.farm.flow_field.x, x)
assert np.allclose(fi.floris.farm.flow_field.y, y)
assert np.allclose(fi.floris.farm.flow_field.z, z)
```

Produces the following output:
```
Using default gauss deflection multipler of 1.2
reinit with res and bounds
/Users/equon/floris/floris/simulation/wake_deflection.py:218: RuntimeWarning: invalid value encountered in true_divide
  sigma_z0 = D * 0.5 * np.sqrt(uR / (U_local + u0))
/Users/equon/floris/floris/simulation/wake_velocity.py:510: RuntimeWarning: invalid value encountered in true_divide
  sigma_z0 = D * 0.5 * np.sqrt(uR / (U_local + u0))
calc with pts
Traceback (most recent call last):
  File "example_get_points_problem.py", line 38, in <module>
    assert np.allclose(fi.floris.farm.flow_field.x, x)
  File "<__array_function__ internals>", line 6, in allclose
  File "/Users/equon/.local/lib/python3.7/site-packages/numpy/core/numeric.py", line 2159, in allclose
    res = all(isclose(a, b, rtol=rtol, atol=atol, equal_nan=equal_nan))
  File "<__array_function__ internals>", line 6, in isclose
  File "/Users/equon/.local/lib/python3.7/site-packages/numpy/core/numeric.py", line 2260, in isclose
    return within_tol(x, y, atol, rtol)
  File "/Users/equon/.local/lib/python3.7/site-packages/numpy/core/numeric.py", line 2246, in within_tol
    return less_equal(abs(x-y), atol + rtol * abs(y))
ValueError: operands could not be broadcast together with shapes (388,110,30) (194,110,30)
```

**Comment by bayc** [comment]
This functionality is not included in the current FLORIS v3.0.
